cmd/legacy/run.go

```diff
@@ -100,7 +102,26 @@ func Run(v *viper.Viper) {
 		todaygoal.Run(v)
 	}
 
-	log.Debugln("command: heartbeat")
+	if v.IsSet("entity") {
+		log.Debugln("command: heartbeat")
 
-	heartbeatcmd.Run(v)
+		heartbeatcmd.Run(v)
+	}
+
+	if v.IsSet("sync-offline-activity") {
+		log.Debugln("command: sync-offline-activity")
+
+		offlinesync.Run(v)
+	}
+
+	log.Fatalf("One of the following parameters has to be provided: %s", strings.Join([]string{
```

Comment:
Does `log.Fatalf` cause the program to exit with an error code?

Reply:
yes! Docs are [here](https://github.com/sirupsen/logrus#level-logging).